app/backend/src/clj/teet/environment.clj

```diff
@@ -118,13 +119,34 @@
    :eelis {:wms-url (->ssm [:eelis :wms-url] nil)}
    :email {:from (->ssm [:email :from] nil)}})
 
+(defn- describe-all-the-params!
+  "Returns names and other metadata without values"
+  []
+
+  (let [q-params {:op :DescribeParameters :request {:MaxResults 50}}
+        get-first! #(aws/invoke @ssm-client q-params)
+        get-next! #(aws/invoke @ssm-client (assoc-in q-params
+                                                    [:request :NextToken]
+                                                    %))]
+    (loop [results [(get-first!)]]
+      (if-let [next-token (:NextToken (last results))]
+        (do
+          (log/debug "ssm describe-params: got next-token, calling again")
+          (recur (conj results (get-next! next-token))))
+        ;; else
+        (do
+          (log/debug "ssm describe-params: no next-token, returning what we got")
+          (mapcat :Parameters results))))))
+
 (defn- load-ssm-config! [base-config]
   (let [old-config @config
         new-config (merge base-config
                           (resolve-ssm-parameters teet-ssm-config))]
     (when (not= old-config new-config)
       (log/info "Configuration changed")
-      (reset! config new-config))))
+      (reset! config new-config)
+      ;; do after config reset, don't block other config reload in case of errors
+      (file-model/apply-attachment-policy! (map :Name (describe-all-the-params!))))))
```

Comment:
I don't understand the rationale for `describe-all-the-params!` why isn't the allowed file suffix list a regular parameter like others?

Reply:
seems more difficult to configure instead of just having a comma separated list of suffixes that are allowed